docs/topics/control-flow.md

```diff
@@ -6,30 +6,57 @@ In Kotlin, `if` is an expression: it returns a value.
 Therefore, there is no ternary operator (`condition ? then : else`) because ordinary `if` works fine in this role.
 
 ```kotlin
-var max = a 
-if (a < b) max = b
+val tax: Int
+val value = 50
```

Comment:
Let's rename "value" to a more meaningful name. For example, income? It's not good to call variables "value"/"variable" and so on; it makes code hard readable.

Reply:
Would "productValue" be acceptable?

Reply:
Let's pause this review for a bit: let's discuss with the team our approach to writing examples.